scripts/deploy/medic-deploy.md

```diff
@@ -0,0 +1,76 @@
+## Medic Deployment
+
```

Comment:
We should add an intro here for folks who have never used these tools or it's been a long time since they have.  Suggesting something, but it's just a suggestion!

```suggestion
## Medic Deployment

This readme talks about how to use `helm` and `kubect`  running locally on your workstation to create, update and delete instances of the CHT User Management tool.  These will be running in Medic's [EKS](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) and these instructions are meant to be followed by Medic teammates who have access to EKS.

General public is welcome to look at these instructions for who they might use them in their own infrastructure. 

### Key/Value pairs used 
```